core/src/stored/stored.cc

```diff
@@ -179,10 +189,14 @@ int main(int argc, char* argv[])
         prt_kaboom = true;
         break;
 
-      case 'p': /* proceed in spite of I/O errors */
+      case 'i': /* proceed in spite of I/O errors */
```

Comment:
```suggestion
      case 'i': /* ignore I/O errors */
```

Reply:
can't we just remove that abomination of a command-line option?